README.md

```diff
@@ -1,28 +1,33 @@
 # AWSP - CLI To Manage your AWS Profiles!
-AWSP provide a interactive terminal to interact with your AWS Profile. The aim of this project is to make it easier to navigate, observe and manage your AWS Profile in the wild. 
+AWSP provides an interactive terminal to interact with your AWS Profiles. The aim of this project is to make it easier to navigate, observe and manage your AWS Profiles in the wild. 
 
 ---
-
+[![codecov](https://codecov.io/gh/kubeopsskills/awsp/branch/beta/graph/badge.svg?token=5VBFYN3BBH)](https://codecov.io/gh/kubeopsskills/awsp)
 [![release](https://img.shields.io/github/v/release/kubeopsskills/awsp?logo=awsp)](https://github.com/kubeopsskills/awsp/releases)
 [![License Apache](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/kubeopsskills/awsp/blob/beta/LICENSE-APACHE)
 [![License MIT](https://img.shields.io/badge/license-MIT-green?label=License)](https://github.com/kubeopsskills/awsp/blob/beta/LICENSE-MIT)
 [![Download](https://img.shields.io/github/downloads/kubeopsskills/awsp/total)](https://github.com/kubeopsskills/awsp/releases)
 
 ---
 
-
 ## Demo
 ![screenshot1](./assets/images/select-profile.png)
 ![screenshot2](./assets/images/select-region.png)
 
-## Prerequisite
+## Installation
+
+AWSP is available on Linux, ARM, macOS and Windows platforms.
+- Binaries for Linux, ARM, Windows and Mac are available as tarballs in the [release](https://github.com/kubeopsskills/awsp/releases) page
```

Comment:
[this comment](https://github.com/kubeopsskills/awsp/pull/2#pullrequestreview-724384330) for this line